sdks/java/io/rabbitmq/build.gradle

```diff
@@ -31,6 +31,8 @@ dependencies {
   testCompile "org.apache.qpid:qpid-broker-core:7.1.5"
   testCompile "org.apache.qpid:qpid-broker-plugins-memory-store:7.1.5"
   testCompile "org.apache.qpid:qpid-broker-plugins-amqp-0-8-protocol:7.1.5"
+  testCompile library.java.hamcrest_core
```

Comment:
Other IOs have these. They make for nicer assertions in unit tests (`assertThat(foo, isLike(some other thing))`

Reply:
💯 hamcrest and assertj make testing in java so much nicer